submission/distributor_test.go

```diff
@@ -28,6 +30,20 @@ import (
 	"github.com/google/certificate-transparency-go/testdata"
 )
 
+var (
+	RootsCert = map[string][]string{
```

Comment:
```suggestion
	RootCerts = map[string][]string{
```
nit: This is a map to a list of certs, so using the plural "Certs" in the name is a little clearer.

Reply:
Done